frontend/src/components/core/Block/styled-components.ts

```diff
@@ -74,12 +93,14 @@ export const StyledElementContainer = styled.div<StyledElementContainerProps>(
 
 interface StyledColumnProps {
   weight: number
+  gap: string
 }
 
 export const StyledColumn = styled.div<StyledColumnProps>(
-  ({ weight, theme }) => {
+  ({ weight, gap, theme }) => {
     const percentage = weight * 100
```

Comment:
Note: both the `HorizontalBlock` (parent element) and each of the `StyledColumn`s need to be aware of the specified gap